docs/5/5.2-jenkins.md

```diff
@@ -12,24 +12,23 @@ You should already be familiar with using the Jenkins web GUI interface however
 
 ## Exercise
 
-1. Setup a fresh Jenkins server.
-2. Enable the Jenkins SSH server *Configure Global Security* / *SSH Server* and set SSHD Port to an unused port such as 2222. You may also need to make the port accessible to your localhost by port forwarding from your virtual machine or container and opening up the port in the firewall.
-3. Add your public ssh key to your Jenkins user's configuration (*Manage Users* / YOUR USER / *Configure* / *SSH Public Keys*).
-4. Download the [Jenkins' java CLI client](https://jenkins.io/doc/book/managing/cli/) from your local server.
-5. Run the help command `java -jar jenkins-cli.jar -s JENKINS_URL -ssh -user USERNAME help` to look at the available commands.
-6. Install the GitHub plugin with the Jenkins CLI.
-7. Update Jenkins plugins using the Jenkins CLI. Note: You may need to get creative here :)
-8. Clone and fork the spring-pet-clinic [project](https://github.com/liatrio/spring-petclinic).
-9. Download [petclinic-simple-config.xml](files/petclinic-simple-config.xml).
-10. Update the url field in the file to point to your Spring Pet Clinic fork.
-11. Create the Jenkins job and verify it was created.
+1. Setup a fresh Jenkins server with the recommended plugins.
+  * ensure you have jdk and Maven installed in your container
+ 
+ ?> Using the [official Jenkins image](https://hub.docker.com/r/jenkins/jenkins) will allow you to install plugins as part of the Docker build process.
+2. Create a `.yaml` file that configures a basic Jenkins instance and the recommended plugins installed in the previous step.
```

Comment:
I think this should mention that the yaml configuration is specifically working with the Jenkins CasC plugin

Reply:
Specified that the .yaml file is for the Jenkins CasC plugin and linked to documentation